migrations/2024-03-06-123525_add_expiry_notification_at/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Introduce a new column `expiry_notification_at` in the `api_tokens` table.
+-- This column will hold the timestamp of when the user was informed about their token's impending expiration.
+ALTER TABLE api_tokens ADD expiry_notification_at TIMESTAMP;
+
```

Comment:
```suggestion
ALTER TABLE api_tokens ADD expiry_notification_at TIMESTAMP;

COMMENT ON COLUMN api_tokens.expiry_notification_at IS 'timestamp of when the user was informed about their token's impending expiration';
```

Reply:
Updated.

Reply:
looks like I missed an `'` in the comment which caused a syntax error 🙈